src/com/theironyard/novauc/User.java

```diff
@@ -0,0 +1,33 @@
+package com.theironyard.novauc;
+
+import java.util.ArrayList;
+
+/**
+ * Created by Eric on 3/5/17.
+ */
+public class User {
+
+    String name;
+    String password;
+    ArrayList<Message> messageList = new ArrayList();
+                /* key, value = user, password */
+    //is this where I would put String password;  ????? or should I make another method in the User class?
+
+    public User(String name, String password, ArrayList messageList) {
+        this.name = name;
+        this.password = password;
+        this.messageList = messageList;
+
+
+
+    }
+
+    public static void storeText(String text) {
+       //  messagelist //somehow I need to create a method that will allow me to store messages.
+
```

Comment:
The logic here would be work like this.  Use text to create a new message object. Then add that new message to the user arraylist messageList.